src/main/java/gift/repository/MemberJpaRepository.java

```diff
@@ -0,0 +1,17 @@
+package gift.repository;
+
+import gift.entity.Member;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Optional;
+
+interface MemberJpaRepository extends JpaRepository<Member, Long> {
+
+    @Query("SELECT m FROM Member m WHERE m.email.value = :email")
+    Optional<Member> findByEmailValue(@Param("email") String email);
+
+    @Query("SELECT COUNT(m) > 0 FROM Member m WHERE m.email.value = :email")
+    boolean existsByEmailValue(@Param("email") String email);
+}
```

Comment:
```suggestion
@Repository
public interface MemberRepository extends JpaRepository<Member, Long> {
    boolean existsByEmail(Email email);
    Optional<Member> findByEmail(Email email);
}
```
JPA를 활용할 때는 이런식으로 사용할 수 있어요! 
명명규칙만 잘 지키면 기본적인 메서드들에 대한 구현을 자동으로 해주기 때문이에요!
구현을 알아서 해주니, Impl은 제외하고, 인터페이스만 남게 된다면 MemberRepository도 필요 없어지고 MemberJpaRepository만 사용할 수 있게 될 거예요.

이런식으로 모든 Repository를 JPA의 기능을 활용하도록 수정해보세요~
단 먼저 JPA가 어떻게, 왜 이런 기능을 제공하는지 알아보셔야 합니다!